### PR TITLE
Adding limit statement feature

### PR DESCRIPTION
**Pull request by rasputino**
I'm using this project to connect to an Access database, that supports the limit statement via "TOP" syntax. The limit is useful for some Linq methods like "Any, First..."

### CODE REVIEW

**Comment by rasputino** [comment]
Is anybody out there?

**Comment by HEskandari** [comment]
I've been meaning to get to this for a few weeks but couldn't because of a few dramas. I'm looking into it now.

**Comment by HEskandari** [comment]
@rasputino did you look into the failing tests?

**Comment by hazzik** [comment]
@HEskandari the tests are failing because `Thing` table is not available for all tests, but `InsertSampleData` executes for all tests

File: src/NHibernate.JetDriver.Tests/JetTestBase.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ protected JetTestBase(bool autoCreateTables)
                 .SetProperty(Environment.ConnectionProvider, typeof(DriverConnectionProvider).FullName)
                 .SetProperty(Environment.ShowSql, "true")
                 .SetDefaultNamespace("NHibernate.JetDriver.Tests.Entities")
-                .SetProperty(Environment.ConnectionString, string.Format("Provider=Microsoft.ACE.OLEDB.12.0;Data Source={0};", DataFile));
+                .SetProperty(Environment.ConnectionString, string.Format("Provider = Microsoft.Jet.OLEDB.4.0;Data Source={0};", DataFile));
```
Comment (HEskandari) [inline src/NHibernate.JetDriver.Tests/JetTestBase.cs]: This is not needed. This addresses the test run failures on x64 environments, but all that's needed to do is to change the Test project build from Any CPU to x86.

**Comment by rasputino** [comment]
Sorry because I'm answering late, I have been busy.

As @hazzik says is because Thing table is not available in the first pass (it's created then) but the tests pass correctly.

http://imgur.com/a/yEnPP

**Comment by HEskandari** [comment]
Closing this one as the changes are already merged in.

**Pull request closed without merging**